src/utils/api/account/lsk.js

```diff
@@ -79,15 +80,21 @@ export const getAccount = async ({
       params: normParams,
     });
 
-    return response.data[0];
+    if (response.data[0]) {
+      const account = { ...response.data[0] };
+      const isAccountUninitialized = !account.summary?.publicKey;
+      if (isAccountUninitialized) {
+        const publicKey = params.publicKey ?? extractPublicKey(params.passphrase);
```

Comment:
If the method is called with address and it's not initialized, this will throw.

Reply:
This is fine. We can not convert the address to public key or passphrase ourselves. The consumer of the method needs to be aware of this. Thats why we throw an error at the end of the function.